rotkehlchen/chain/evm/decoding/monerium/constants.py

```diff
@@ -0,0 +1,6 @@
+
+from typing import Final
+
+
+CPT_MONERIUM: Final = 'monerium'
+BURN_MONERIUM_SIGNATURE: Final = bytearray.fromhex('3823caec')
```

Comment:
Instead of doing it like this calling the function write the output since is constant